packages/ui/src/components/va-dropdown/VaDropdown.demo.vue

```diff
@@ -113,6 +113,9 @@
         <va-dropdown
           anchor-selector="#anchor-selector"
         >
+          <template #anchor>
```

Comment:
Should work without #anchor slot, because you have anchor-selector. Maybe remove warning and let it be empty.